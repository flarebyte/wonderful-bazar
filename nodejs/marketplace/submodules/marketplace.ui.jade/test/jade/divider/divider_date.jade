include mixins/divider

ul
  mixin divider_date(everything.title, everything)
  mixin divider_count(no_modified.title, no_modified)
