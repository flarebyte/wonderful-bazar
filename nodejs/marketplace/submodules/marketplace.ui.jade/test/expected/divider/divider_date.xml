<ul><li data-role="list-divider" data-theme="f">Users Title<span class="pull-right"><meta itemprop="dateModified" content="2012-10-11T01:10:00Z"/><span class="label info">11</span><span>&nbsp;</span><span class="label info">October</span><span>&nbsp;</span><span class="label info">2012</span></span></li><li data-role="list-divider" data-theme="f">Users Title<span class="ui-li-count"></span></li></ul>